homeassistant/components/heos/config_flow.py

```diff
@@ -21,6 +21,11 @@ class HeosFlowHandler(config_entries.ConfigFlow):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
 
+    async def async_step_discovery(self, discovery_info):
```

Comment:
Discovery is usually placed on the bottom, together with import

Reply:
I like to keep things alphabetically sorted.  You need this moved?

Reply:
I don't think it is mandatory